tests/ryujit_x86_no_fallback_issues.targets

```diff
@@ -20,7 +20,7 @@
             <Issue>2235</Issue>
```

Comment:
I'm going to post a PR to remove this file as soon as you merge, so feel free to revert the change to this file.